### PR TITLE
libbotan: Disable thread_utils.

### PR DESCRIPTION
**Pull request by dewyatt**
See https://github.com/randombit/botan/issues/2582.

### CODE REVIEW

File: mingw-w64-libbotan/PKGBUILD [hunk 1]
```diff
@@ -59,6 +59,7 @@ build() {
   MSYS2_ARG_CONV_EXCL="--prefix=;--libdir=;--bindir=" \
```
Comment (lazka) [inline mingw-w64-libbotan/PKGBUILD:59]: Please add a small comment here linking the the upstream discussion, and/or why `-disable-modules=thread_utils` is passed. So we know why it's there in the future.

**Comment by lazka** [comment]
increase pkgrel

**Comment by dewyatt** [comment]
Roger that, updated (squashed it down -- hope that's ok).

**Comment by dewyatt** [comment]
We can grab the key from https://botan.randombit.net/pgpkey.txt if desired btw

**Comment by lazka** [comment]
yeah, but pacman doesn't support that. Maybe we can think about adding those keys to the repo though.. not sure.

**Pull request merged**